pkgs/development/python-modules/pandantic/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, multiprocess
+, pandas
+, pandas-stubs
+, poetry-core
+, pydantic
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pandantic";
+  version = "0.3.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "wesselhuising";
+    repo = "pandantic";
+    rev = "f0d749459876a17bd5db53a69ffdfe3bbfc197f5";
+    hash = "sha256-RXonAk9iy+SFvYtkI2AFIZBogIUDmQfUwKc1Da13viM=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    multiprocess
+    pandas
+    pandas-stubs
+    pydantic
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [
+    "pandantic"
+  ];
+
+  disabledTests  = [
+    "test_read_with_lxml_dtype_backend"
+  ];
+
+  meta = with lib; {
+    description = "About
+Enriches the Pydantic BaseModel class by adding the ability to validate dataframes using the schema and custom validators of the same BaseModel class.";
+    homepage = "https://github.com/wesselhuising/pandantic";
+    maintainers = with maintainers; [ yannip ];
```

If there is no license statement from upstream, the module has to be considered `unfree`.

```suggestion
    # https://github.com/wesselhuising/pandantic/issues/19
    license =licenses.unfree;
    maintainers = with maintainers; [ yannip ];
```

Perhaps we get a quick response: https://github.com/wesselhuising/pandantic/issues/19

---

I've made this unfree for now. Seems like no answer yet